internal/services/managedhsm/key_vault_managed_hardware_security_module_resource_test.go

```diff
@@ -186,13 +187,14 @@ provider "azurerm" {
 %s
 
 resource "azurerm_key_vault_managed_hardware_security_module" "test" {
-  name                     = "kvHsm%d"
-  resource_group_name      = azurerm_resource_group.test.name
-  location                 = azurerm_resource_group.test.location
-  sku_name                 = "Standard_B1"
-  tenant_id                = data.azurerm_client_config.current.tenant_id
-  admin_object_ids         = [data.azurerm_client_config.current.object_id]
-  purge_protection_enabled = false
+  name                          = "kvHsm%d"
+  resource_group_name           = azurerm_resource_group.test.name
+  location                      = azurerm_resource_group.test.location
+  sku_name                      = "Standard_B1"
+  tenant_id                     = data.azurerm_client_config.current.tenant_id
+  admin_object_ids              = [data.azurerm_client_config.current.object_id]
+  purge_protection_enabled      = false
+  public_network_access_enabled = false
```

Comment:
Same here, from the name of the config this is only checking whether the required properties are updateable.
```suggestion
```